ui/src/main/js/component/stock-list/stock-details.js

```diff
@@ -1,4 +1,4 @@
-import Chart from './chart.js';
+import Chart from '../chart/index.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7238741)
```suggestion
import Chart from "../chart/index.js";
```